src/Agent/NewRelic/Agent/Core/AgentHealth/AgentHealthReporter.cs

```diff
@@ -829,5 +830,17 @@ protected override void OnConfigurationUpdated(ConfigurationUpdateSource configu
             // Some one time metrics are reporting configured values, so we want to re-report them if the configuration changed
             _oneTimeMetricsCollected = false;
         }
+
+        private void ReportIfLoggingDisabled()
+        {
+            if (!_configuration.LoggingEnabled)
+            {
+                ReportSupportabilityCountMetric(MetricNames.SupportabilityLoggingDisabled);
```

Comment:
Is the name of this metric and corresponding methods clear enough to differentiate them from log forwarding and decoration? I'm not sure if referring to these things as Agent Logs helps with disambiguating the different "Logs".

Reply:
Yeah I struggled a bit with the names. If anyone has a better suggestion I'd be glad to hear it. "Internal logs"?

Reply:
I think "AgentLogging" is clear enough; we refer to the instrumentation for customers' application logs as "ApplicationLogging" in several places (in config, anyway, perhaps not in supportability metrics).